extensions/wikia/Chat2/js/controllers/controllers.js

```diff
@@ -386,6 +388,13 @@ var NodeRoomController = $.createClass(Observable, {
 		}), 10000);
 	},
 
+	onLongMessage: function (message) {
+		if (typeof message.user == 'string' && message.user == wgUserName) {
```

Comment:
You can use `===` operator that will check type as well, hence `typeof` won't be needed (assuming `wgUserName` is a string :) ).